stack.yaml

```diff
@@ -1,10 +1,10 @@
-name: Node.js Cloud Events
+name: JavaScript FaaS
 version: 0.0.1
-description: Serverless Node.js function runtime for Cloud Events
+description: Node.js Function Runtime
 license: Apache-2.0
-language: nodejs
+language: JavaScript
 maintainers:
-  - name: Lance Ball
-    email: lball@redhat.com
-    github-id: lance
```

Comment:
If you don't mind leaving me in as a maintainer, I'd appreciate it. :)

Reply:
Thank you, I wasn't sure what the expectation was there and am happy to leave you the designated maintainer.

Reply:
I didn't mean to imply that _you_ shouldn't be there as well :)